src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -619,6 +616,13 @@ private async Task RemoveEventHandlerIdsAsync(ArrayRange<int> eventHandlerIds, T
                 }
             }
             else
+            {
+                _ = ContinueAfterTask(eventHandlerIds, afterTaskIgnoreErrors);
+            }
+
+            // Factor out the async part into a separate local method purely so, in the
+            // synchronous case, there's no state machine or task construction
+            async Task ContinueAfterTask(ArrayRange<int> eventHandlerIds, Task afterTaskIgnoreErrors)
```

Comment:
`static async Task`?

Reply:
It can't be static because it calls an instance method.